src/graph/backend/dnnl/passes/constant_propagation.cpp

```diff
@@ -34,18 +34,12 @@ namespace {
 bool has_scratchpad(const op_t *op) {
     // the following ops do not have scratchpad output by definition
     const static std::set<op_kind_t> no_scratchpad_ops {
-            op_kind::dnnl_constant_scales,
-            op_kind::dnnl_constant_zps,
-            op_kind::dnnl_add_zps,
-            op_kind::dnnl_sub_zps,
-            op_kind::dnnl_to_group,
-            op_kind::dnnl_from_group,
-            op_kind::dnnl_permute,
-            op_kind::dnnl_squeeze,
-            op_kind::dnnl_unsqueeze,
-            op_kind::dnnl_transpose,
-            op_kind::dnnl_reshape,
-    };
+            op_kind::dnnl_constant_scales, op_kind::dnnl_constant_zps,
+            op_kind::dnnl_add_zps, op_kind::dnnl_sub_zps,
+            op_kind::dnnl_to_group, op_kind::dnnl_from_group,
+            op_kind::dnnl_permute, op_kind::dnnl_squeeze,
+            op_kind::dnnl_unsqueeze, op_kind::dnnl_transpose,
+            op_kind::dnnl_reshape, op_kind::dnnl_gen_index, op_kind::dnnl_mask};
```

Comment:
Nit: I'd add a comma at the end to keep previous formatting.

Reply:
reformatted, thanks